panopticum/models.py

```diff
@@ -301,8 +307,10 @@ def __str__(self):
 class ComponentVersionModel(models.Model):
     component = models.ForeignKey(ComponentModel, on_delete=models.PROTECT, related_name='component_version')
 
-    version = models.CharField(max_length=64, verbose_name="Version or build",
-                               help_text="note: component version instance will be cloned if you change version!")
+    version = models.ForeignKey(ComponentVersionListModel, to_field='version',
```

Comment:
I suggest to leave ```models.CharFiled``` and move ```release_date``` field to ```ComponentVersion```. My opinion ```ComponentVersionList``` equal to ```ComponentVersion```. Also we should review ```meta_updated``` field: may be it equal to ```release_time```. @perfguru87 ?